traits/_py2to3.py

```diff
@@ -12,16 +12,10 @@
 else:
     LONG_TYPE = int
 
-if sys.version_info[0] < 3:
-    import string
-
-    str_find = string.find
-    str_rfind = string.rfind
-else:
-    str_find = str.find
-    str_rfind = str.rfind
+str_find = str.find
```

Comment:
Let's get rid of these. I didn't find any uses of them in this codebase (and there shouldn't be any uses in other codebases, since this is a private module).

Reply:
alas - https://github.com/enthought/traitsui/pull/496

Reply:
Okay, in that case, let's leave these names in for now; we don't want Traits master to break released Traits UI if we can help it. Please could you add a comment to the effect that these constants are still being used by Traits UI, and a link to the PR you just made?

Reply:
We should also check `PyFace` and other ETS modules for uses of `traits._py2to3`.

Reply:
reverted the change. don't find any use of the private module in pyface, apptools, mayavi.